### PR TITLE
lori wants the table to fade while filtering is loading because for m…

### PR DESCRIPTION
**Pull request by jasonfine**
…ore intense logic it appears that nothing is happening or something is borked to the user

### CODE REVIEW

File: app/assets/javascripts/fae/form/_filtering.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ Fae.form.filtering = {
     // hardcode _this because this === Fryr object
     var _this = Fae.form.filtering;
     var post_url = _this.$filter_form.attr('action');
+    $('.js-sort-row').css({opacity: 0.5});
```
Comment (jamesmk) [inline app/assets/javascripts/fae/form/_filtering.js]: @jasonfine I'd rather not apply CSS with JS. Can you apply a class with JS and use CSS on the class?

thanks!

**Comment by jamesmk** [comment]
@jasonfine Can you add a changelog item for this as well?

**Comment by jasonfine** [comment]
all set

**Comment by jamesmk** [comment]
@jasonfine Oops, conflict from #144. Fix that up and we're good to go!

**Pull request merged**